gulpfile.js

```diff
@@ -94,10 +94,9 @@ gulp.task('lint', ['build'], function () {
     'use strict';
 
     return gulp.src(paths.sourceFiles)
-        .pipe(react())
```

Comment:
Our CI indicates that everything is working properly. Our lint task is isolated from the rest of the build process so it shouldn't really have any other impacts. However, I did do some manual testing anyway and everything appears to be in order.